src/repo.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * All repo relate configuration lives within this file. Add domains,
+ * effects, and configure any repo startup behavior here.
+ */
+
+import Microcosm from 'microcosm'
+import Game from './domains/game'
+import Timer from './effects/timer'
+
+export default class Repo extends Microcosm {
```

Comment:
ah of course, i didnt even see the microcosm extend portion of this class declaration. so all the `repo.push()` calls i'm seeing aren't so much triggering an action to the game, it's more pushing a state to microcosm, which in turn triggers an action to the game?